dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -870,4 +872,6 @@ portalRouter.route('/cron-jobs').delete(cronJobsController.deleteAllEstoreLogs);
 portalRouter.route('/gef/mandatory-criteria/latest').get(mandatoryCriteria.getLatestGefMandatoryCriteria);
 portalRouter.route('/gef/mandatory-criteria/version/:version').get(mandatoryCriteria.getGefMandatoryCriteriaByVersion);
 
+portalRouter.route('/previous-reports/:bankId').get(previousReports.getUtilisationReports);
```

Comment:
Can we add a validation on the bankId here too? it'll be the same as in portal API just checking the format of it